src/me/StevenLawson/TotalFreedomMod/Commands/Command_blockcmd.java

```diff
@@ -8,7 +8,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
-@CommandParameters(description = "Block all commands for a specific player.", usage = "/<command> <purge | <partialname>>", aliases = "blockcommands,blockcommand")
+@CommandParameters(description = "Block all commands for a specific player.", usage = "/<command> [purge] | [partialname] [-a]>", aliases = "blockcommands,blockcommand")
```

Comment:
This is not exactly what I meant. This way, it is suggested `purge`, `partialname` and `-a` can be used independently of one another, but all three are optional. This is not quite right. The correct line should be.

`"/<command> <-a | purge | partialname>"